tensorflow_lite_support/examples/task/text/desktop/python/README.md

```diff
@@ -68,5 +68,66 @@ category[0]: 'Negative' : '0.81313'
 category[1]: 'Positive' : '0.18687'
 ```
 
+## BertNLClassifier
+
+#### Prerequisites
+
+You will need:
+
+*   a Bert based TFLite text classification model from model maker. (e.g.
+    [movie_review_model][3] available on TensorFlow Hub).
+
+#### Usage
+
+First, download the pretrained model by:
+
+```bash
+curl \
+ -L 'https://url/to/bert/nl/classifier' \
+ -o /tmp/bert_movie_review.tflite
+```
+
+You can either use the prebuilt demo tool from TFLite Support Pypi package or
+build from source. See the instruction below.
+
+##### Access from TFLite Support Pypi package
+
+Install the Support Pypi package by:
+
+```bash
+pip install tflite-support-nightly
+```
+
+Run the demo tool:
+
+```bash
+bert_nl_classifier_demo  \
+--model_path=/tmp/bert_movie_review.tflite \
+--text="it's a charming and often affecting journey"
+```
```

Comment:
Please remove this part first, and remove the one for NLClassifier as well. Thanks.

Reply:
Hi @lu-wang-g! I just updated the README